src/ketView.tsx

```diff
@@ -8,31 +8,45 @@ export const VIEW_TYPE_KET = "ket-view";
 
 export class KetView extends TextFileView {
 	ketcher: Ketcher;
+	subscriber: any;
 
 	getViewData() {
 		return this.data;
 	}
 
-	// If clear is set, then it means we're opening a completely different file.
 	setViewData(data: string, _clear: boolean) {
 		this.data = data;
 
-		ReactDOM.unmountComponentAtNode(this.containerEl.children[1]);
-		const container = this.containerEl.children[1];
-		ReactDOM.render(
-			<React.StrictMode>
-				<KetcherReact
-					data={this.data}
-					onInit={(ketcher: Ketcher) => {
-						this.ketcher = ketcher;
-						// https://github.com/epam/ketcher/issues/2250
-						// @ts-ignore
-						global.ketcher = ketcher;
-					}}
-				/>
-			</React.StrictMode>,
-			container
-		);
+		// If clear is set, then it means we're opening a completely different file.
+		if (_clear) {
+			this.ketcher?.editor.unsubscribe("change", this.subscriber);

```

Comment:
Avoid unsubscribe before editor initialize (will happen only when we open a new ketcher view)